### PR TITLE
V72.2 Data model

### PR DESCRIPTION
**Pull request by barry-jones50410**
v72_2 schema, including necessary changed to the security and index scripts. ERD also updated.

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/db/reference/modernised_darts_schema.sql [hunk 1]
```diff
@@ -1041,6 +1060,25 @@ IS 'foreign key from transcription_document';
 COMMENT ON COLUMN external_object_directory.ado_id
 IS 'foreign key from annotation_document';
 
+CREATE TABLE extobjdir_process_detail
```
Comment (hemantasharma1129) [inline src/main/resources/db/reference/modernised_darts_schema.sql:1063]: @jackmaloney This is a new table to hold 2 of the EOD fields for now. Though there is no Dev tickets yet to make the corresponding code in dart-api.

Comment (jackmaloney) [reply src/main/resources/db/reference/modernised_darts_schema.sql:1063]: Shall we just put a ticket on the backlog and leave Joe to prioritise it?

**Review by jackmaloney — APPROVED** [review]

**Review by hemantasharma1129 — APPROVED** [review]

File: src/main/resources/db/reference/modernised_darts_schema.sql [hunk 2]
```diff
@@ -1041,6 +1060,25 @@ IS 'foreign key from transcription_document';
 COMMENT ON COLUMN external_object_directory.ado_id
 IS 'foreign key from annotation_document';
 
+CREATE TABLE extobjdir_process_detail
+(epd_id                      INTEGER                       NOT NULL
+,eod_id                      INTEGER                       NOT NULL          UNIQUE
+,event_date_ts               TIMESTAMP WITH TIME ZONE
+,update_retention            BOOLEAN                       NOT NULL
+,created_ts                  TIMESTAMP WITH TIME ZONE      NOT NULL
+,created_by                  INTEGER                       NOT NULL
```
Comment (chrisbellingham-hmcts) [inline src/main/resources/db/reference/modernised_darts_schema.sql:1069]: Needs a FK constraint

File: src/main/resources/db/reference/modernised_darts_schema.sql [hunk 3]
```diff
@@ -1041,6 +1060,25 @@ IS 'foreign key from transcription_document';
...
 
+CREATE TABLE extobjdir_process_detail
+(epd_id                      INTEGER                       NOT NULL
+,eod_id                      INTEGER                       NOT NULL          UNIQUE
+,event_date_ts               TIMESTAMP WITH TIME ZONE
+,update_retention            BOOLEAN                       NOT NULL
+,created_ts                  TIMESTAMP WITH TIME ZONE      NOT NULL
+,created_by                  INTEGER                       NOT NULL
+,last_modified_ts            TIMESTAMP WITH TIME ZONE      NOT NULL
+,last_modified_by            INTEGER                       NOT NULL
```
Comment (chrisbellingham-hmcts) [inline src/main/resources/db/reference/modernised_darts_schema.sql:1071]: Needs a FK constraint

File: src/main/resources/db/reference/modernised_darts_schema.sql (hunk 2 above)
Comment (chrisbellingham-hmcts) [reply src/main/resources/db/reference/modernised_darts_schema.sql:1069]: Will be incorporated in v72.3.

**Review by chrisbellingham-hmcts — APPROVED** [review]

**Pull request merged**